.github/workflows/build-test.yml

```
@@ -36,6 +36,7 @@ jobs:
       os-name: linux
       cc: ${{ matrix.cc }}
       asan: ${{ matrix.asan }}
+      gcov: 0
```

```suggestion
      dev-gcov: 0
```

And dev-asan, so we are consistent.